xbmc/cores/RetroPlayer/RetroPlayerTypes.h

```diff
@@ -18,5 +18,18 @@ namespace RETRO
 class IRenderBufferPool;
 using RenderBufferPoolPtr = std::shared_ptr<IRenderBufferPool>;
 using RenderBufferPoolVector = std::vector<RenderBufferPoolPtr>;
+
+enum class DataAccess
+{
+  READ_ONLY,
+  WRITE_ONLY,
```

Comment:
Do you have an example when this may occur?

Reply:
Some kinds of mapped memory can be write-only. Can't think of any off the top of my head, but I clearly had a need for this when I wrote it in 2018 (see in the git log). I was working on a memory maps feature that I never finished.